distributed/ddp-tutorial-series/requirements.txt

```diff
@@ -0,0 +1 @@
+torch>=1.11.0
```

Comment:
since 1.12 is out, should we just use that?

Reply:
I wouldn't want to limit the audience / force an upgrade just for the tutorial... that being said, i'm probably not using anything specific to 1.11 either

Reply:
We can upgrade to the highest version that's better